.golangci.yml

```diff
@@ -17,6 +17,7 @@ run:
   # default value is empty list, but default dirs are skipped independently
   # from this option's value (see skip-dirs-use-default).
   skip-dirs:
+  - third_party
```

Comment:
Is the intention to keep the package as close to upstream as possible so it's easier to sync if/when needed? If that's not the case, why not fix all lint issues and treat the package as any other?

Reply:
I'm not worried about syncing changes from upstream as its a very small amount of imported code which is a pretty simple wrapper around syscalls. I just thought it was standard practice to leave the third party code untouched if possible (e.g. if you look at the comment a few lines below, golang-ci provides the ability to disable linting of third_party by convention, with slightly different semantics).

I'm definitely not against fixing the linting issues if we think that is tidier, but it would change the source files a lot (they make heavy use of all caps variable names, etc).